packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts.tsx

```diff
@@ -168,7 +168,7 @@ const CompareCFDs = observer(() => {
                                 />
                             )}
                             {/* Renders cTrader data */}
-                            {all_cfd_available_accounts.length > 0 && has_ctrader_account_available && (
+                            {all_cfd_available_accounts.length === -2 && has_ctrader_account_available && (
```

Comment:
```suggestion
                            {!all_cfd_available_accounts.length && has_ctrader_account_available && (
```
This is extremely odd 👀  not gonna lie

Reply:
This was to make this always falsy and not to comment it. I make changes for this.